bpf/Makefile

```diff
@@ -12,12 +12,18 @@ BPF = bpf_lxc.o bpf_netdev.o bpf_overlay.o bpf_lb.o bpf_xdp.o
 SCRIPTS = init.sh join_ep.sh run_probes.sh spawn_netns.sh
 LIB := $(shell find ./ -name '*.h')
 
+TARGET=cilium-map-migrate
+
 CLANG ?= clang
 LLC   ?= llc
 
 ifeq ("$(PKG_BUILD)","")
 
-all: $(BPF)
+all: $(BPF) $(TARGET)
+
+$(TARGET): $(TARGET).c
+	@$(ECHO_CC)
+	$(QUIET) $(CC) -Wall -O2 -lelf -I include/ $@.c -o $@
```

Comment:
Is `$(CC)` guaranteed to be defined? The rest of the file uses `$(CLANG)`.

Reply:
Probably the `$(ECHO_CC)` is misleading here, we could rename either `$(CLANG)` or `$(ECHO_CC)` so they're consistent. Minor thing.

Reply:
I've built clang only with bpf target locally, but agree, reasonable to assume that x86 target should be present as well since we also do run_probes.sh with it.

Reply:
In the CI, I'm still seeing the above failure to build and install cilium-map-migrate.